app/src/main/java/com/pysiak/simple/clock/ui/alarm/AlarmView.java

```diff
@@ -0,0 +1,10 @@
+package com.pysiak.simple.clock.ui.alarm;
+
+/**
+ * Defines contract for fragment Alarm.
```

Comment:
Don't use fragment/activity when describe interface of view - it is incorrect.

Instead of this, use something like "Defines contract for alarm screen". Or for alarm tab, if you want....